Code/Source/URDF/Constants.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <string>
+
+namespace ROS2
+{
+    ////////////////////////////////////////////////////////////////////////////////////////////////////
+    //! The struct contains all constant parameters used in FBX files.
+    //! TODO: Could be moved to configuration file
+    namespace Fbx
+    {
+        struct Constants
+        {
+            struct FbxHeader
+            {
+                inline static const int headerVersion = 1003;
+                inline static const int fileVersion = 7500;
+                inline static const std::string creatorName = "O3DE ROS2 Gem";
```

Comment:
If you make the change to use AZStd::string instead of std::string, then you should not use AZStd::string as static variables due to timing of its allocator availability. Instead, you should use string constants instead:
```suggestion
                constexpr char* creatorName = "O3DE ROS2 Gem";
```

Reply:
As above. I'll consider the change to AZStd::string.